dockerfiles/base/scripts/base/library.sh

```diff
@@ -256,21 +256,28 @@ docker_compose() {
   fi
 }
 
+# Return srandom character alphanumeric string (upper and lower case) of length $1
+generate_random_string() {
```

Comment:
I am sure that is typical, but I can see a number of situations in the near future where we may want to use it.  It wasn't the easiest thing to dig up - so is it inappropriate to leave in unused methods?